.gitignore

```diff
@@ -34,3 +34,5 @@
 
 # Ignore the manifest.js
 /app/assets/config/manifest.js
+
+/config/application.rb
```

Comment:
Is this an okay file to ignore?

Reply:
Hmm I'm not sure. I don't think we want to ignore it because it is needed to make the API run probably, correct?

Reply:
Sorry, I'll fix that